setup.py

```diff
@@ -10,16 +10,16 @@
 
 install_requires = [
     'Django>=1.8,<1.12',
-    'djangorestframework==3.6.4', # Latest version that supports both Django 1.8 and 1.11
+    'djangorestframework==3.6.4',  # Latest version that supports both Django 1.8 and 1.11
     'elasticsearch>=2.4.1,<3',
-    'requests>=2.14,<2.15',
-    'urllib3>=1.21,<1.22',
+    'requests>2.19,<2.20',
+    'urllib3>1.21,<1.24',
     'django-localflavor>=1.5,<1.6',
-    'wagtail-flags>=2.0.5,<2.2',
+    'wagtail-flags>=3.0.0,<3.1',
```

Comment:
Any particular reason not to make this `<4.0`? If projects are using semantic versioning can we count on minor releases not breaking things?

Reply:
> If projects are using semantic versioning can we count on minor releases not breaking things? 

Is that a rhetorical question? I guess since we own `wagtail-flags`, it will *probably* be using semantic versioning correctly, but it's not guaranteed.

I'm interpreting your comment to mean that, in general, we should allow any minor version and patch updates for a library, unless we know there's a reason to constrain the versions more. Is that what you mean?

Reply:
Yes, that's what I had in mind, although we can constrain further to patch versions if you'd prefer.